extractor/filesystem/list/list.go

```diff
@@ -282,6 +286,12 @@ var (
 		{gcpoauth2access.NewDetector(), "secrets/gcpoauth2accesstoken", 0},
 	})
 
+	// Credential extractors.
```

Comment:
Start the comment with `// Secrets ...` - our internal linter requires that all public fields have a comment starting with the name of the field.